<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSaveLog</class>
 <widget class="QWidget" name="frmSaveLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTextEdit" name="txtMain"/>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QLineEdit" name="txtPort">
          <property name="text">
           <string>6000</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cboxSize"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cboxRow"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labRow_2">
          <property name="text">
           <string>窗口行数</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labSize">
          <property name="text">
           <string>文件大小</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labRow">
          <property name="text">
           <string>文件行数</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="cboxViewRows"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labPort">
          <property name="text">
           <string>监听端口</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listType"/>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_LogtoFile">
        <property name="text">
         <string>开启日志文件重定向</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_LogtoNet">
        <property name="text">
         <string>日志输出到网络</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_LogtoForm">
        <property name="text">
         <string>开启窗口重定向</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_LogCancel">
        <property name="text">
         <string>取消重定向</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLog">
        <property name="text">
         <string>手动清除窗体日志</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
